ADOPTERS.md

```diff
@@ -36,7 +36,7 @@ including the Balena project listed below.
 
 **_BuildKit_** - The Moby project's [BuildKit](https://github.com/moby/buildkit) can use either runC or containerd as build execution backends for building container images. BuildKit support has also been built into the Docker engine in recent releases, making BuildKit provide the backend to the `docker build` command.
 
-**_Azure acs-engine_** - Microsoft Azure's [acs-engine](https://github.com/Azure/acs-engine) open source project has customizable deployment of Kubernetes clusters, where containerd is a selectable container runtime. At some point in the future Azure's AKS service will default to use containerd as the CRI runtime for deployed Kubernetes clusters.
+**_[Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service)_** - Microsoft's managed Kubernetes offering  has used containerd as the default container runtime for Linux nodes since v1.19. Containerd for Windows nodes is currently in public preview. [More Details](https://docs.microsoft.com/en-us/azure/aks/cluster-configuration#container-runtime-configuration)
```

Comment:
```suggestion
**_[Azure Kubernetes Service (AKS)](https://azure.microsoft.com/services/kubernetes-service)_** - Microsoft's managed Kubernetes offering  has used containerd as the default container runtime for Linux nodes since v1.19. Containerd for Windows nodes is currently in public preview. [More Details](https://docs.microsoft.com/azure/aks/cluster-configuration#container-runtime-configuration)
```

Reply:
is it default since 1.19 or the only?

Reply:
It is has been default since 1.19 for linux. It is also now the only runtime. Let me check the last K8s version users could still use docker with (for linux).

Reply:
It is the only runtime since v1.19 :) I updated the PR.